Khan/wonder-blocks
# Add Storybook controls for TextField

## Summary:
- Add storybook controls to Textfield via argtypes file
- Add documentation for each story
- Move .stories.js file under __docs__ directory
- Change TextField `id`s so they're not duplicated in order to remove the console warnings

Issue: https://khanacademy.atlassian.net/browse/WB-1294

## Test plan:
- Go to http://localhost:6061/?path=/docs/form-textfield--default
- Go to the Docs tab
- Confirm that all the props table is completely filled in
- Go to the Canvas tab
- Confirm that the controls work for the Default story
- Confirm that the behvaior works as expected for all other text fields
- Reread all the story descriptions

## Images:
<img width="1110" alt="Screen Shot 2022-05-23 at 5 26 21 PM" src="https://user-images.githubusercontent.com/13231763/169924898-c32f43fc-9dd0-42f1-b725-b4f98875c597.png">
<img width="1068" alt="Screen Shot 2022-05-23 at 5 26 33 PM" src="https://user-images.githubusercontent.com/13231763/169924912-338cdb22-78bb-465c-8b53-32dbf77979b9.png">
<img width="1064" alt="Screen Shot 2022-05-23 at 5 26 43 PM" src="https://user-images.githubusercontent.com/13231763/169924919-56497fe6-4d0d-48fe-a67f-0755cd07f327.png">
<img width="1046" alt="Screen Shot 2022-05-23 at 5 26 50 PM" src="https://user-images.githubusercontent.com/13231763/169924931-1ec392fb-ada3-4130-abec-a12c11b09a40.png">
<img width="1049" alt="Screen Shot 2022-05-23 at 5 26 57 PM" src="https://user-images.githubusercontent.com/13231763/169924944-bf1a1f67-ec38-4d7b-8f06-2abedbfb0fe7.png">
<img width="1059" alt="Screen Shot 2022-05-23 at 5 27 06 PM" src="https://user-images.githubusercontent.com/13231763/169924959-789d45b2-74a2-42b4-9391-51db2e862d37.png">
<img width="1070" alt="Screen Shot 2022-05-23 at 5 27 15 PM" src="https://user-images.githubusercontent.com/13231763/169924963-3f1b4a0a-5316-4de0-b40f-1a29805c693c.png">
